gnosis/protocol/tests/test_gnosis_protocol_api.py

```diff
@@ -14,7 +14,8 @@ class TestGnosisProtocolAPI(TestCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.gnosis_protocol_api = GnosisProtocolAPI(EthereumNetwork.RINKEBY)
+        cls.gnosis_protocol_api = GnosisProtocolAPI(EthereumNetwork.GOERLI)
```

Comment:
Could be goerli_gnosis_protocol_api? It's a little bit confused have mainnet and not goerli.